data/newMigrations/017-service_entries.js

```diff
@@ -0,0 +1,28 @@
+exports.up = (knex) => {
+  return knex.schema.createTable('service_entries', (tbl) => {
+    tbl
+      .uuid('service_entry_id')
+      .primary()
+      .defaultTo(knex.raw('gen_random_uuid ()'));
+    tbl
+      .integer('service_type_id')
+      .unsigned()
+      .notNullable()
+      .references('service_type_id')
+      .inTable('service_types');
```

Comment:
Don't forget the onUpdate and onDelete!

Reply:
By default onUpdate and onDelete are set to restrict, so it's only actually needed when you want to change to cascade.

Do you still want it added in?

Reply:
Oh!  My bad, that works